turbinia/output_manager.py

```diff
@@ -170,7 +170,7 @@ def save_evidence(self, evidence_, result):
     evidence_.saved_path = path
     evidence_.saved_path_type = path_type
     log.info(
-        'Saved copyable evidence data to {0:s}'.format(evidence_.saved_path))
+        'Saved copyable evidence data to {0!s:s}'.format(evidence_.saved_path))
```

Comment:
I think what you want here and below is just `{0!s}`.  I'm not sure what this format string will evaluate to.

Reply:
Fixed.